infrastructure/proto/log_replication_metadata.proto

```diff
@@ -105,4 +105,11 @@ message ReplicationEvent {
   string clusterId = 1;
   string eventId = 2;
   ReplicationEventType type = 3;
+}
+
+enum ReplicationModel {
+  TABLE_REPLICATION = 0;          // Full table replication (Single Source to Single Sink = 1:1)
+  ROUTING_QUEUES = 1;             // Routing queue replication (used for entry-level replication) (1:1)
+  LOGICAL_GROUP_REPLICATION = 2;  // Table association to logical group (1:1)
+  MERGE_MULTI_TABLES = 3;         // Same table replication from multiple sources to single sink (n:1)
```

Comment:
Will this make the names pattern more consistent？
ROUTING_QUEUES -> ROUTING_QUEUES_REPLICATION
MERGE_MULTI_TABLES -> MULTI_SOURCE_REPLICATION

Reply:
Makes sense, though that would make the names too long and it would read out like ReplicationModel.Routing_Queue_Replication, which brings a good point we can actually remove the 'replication' word for all as its redundant with 'ReplicationModel'. Let me think of a better name for 'Table_replication' which can't just be called table. If you have any suggestions let me know! Based on this suggestion, so far I would keep:
ROUTING_QUEUES, LOGICAL_GROUPS, MULTI_SOURCE_MERGE and (thinking on the first one)

Reply:
The new names sound good to me!

Reply:
We can consider calling it SINGLE_SOURCE_SINK